CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [15.5.3] - 2025-XX-XX
+* Fixed backup errors with USB when file and app backup are both on
+* Fixed overdue backups not automatically starting when USB drives are plugged in
+* Fixed "System apps" not being backed up
+* Fixed a common error (StaleDataException) causing backups to fail
+* Updated dependencies for 15 QPR1
```

Comment:
Awesome, please also include https://github.com/seedvault-app/seedvault/pull/849 and https://github.com/seedvault-app/seedvault/pull/857 which we want to get into this release.